docs/docs/chitchat-faqs.mdx

```diff
@@ -265,76 +85,112 @@ pipeline:
 
-:::note
-Here's a minimal checklist of files we modified to build a basic FAQ assistant:
+### NLU Training Data
 
-* `data/nlu.md`: Add NLU training data for `faq/` intents
+NLU training examples for the `ResponseSelector` look the same as
+regular training examples, except that their names must refer to the retrieval
+intent they are grouped under. For example:
 
-* `data/responses.md`: Add responses associated with `faq/` intents
+```yaml
+nlu:
+  - intent: faq/ask_channels
+    examples: |
+      - What channels of communication does rasa support?
+      - what channels do you support?
+      - what chat channels does rasa uses
+      - channels supported by Rasa
+      - which messaging channels does rasa support?
+  - intent: faq/ask_languages
+    examples: |
+      - what language does rasa support?
+      - which language do you support?
+      - which languages supports rasa
+      - can I use rasa also for another laguage?
+      - languages supported
+  - intent: faq/ask_rasax
+    examples: |
+      - I want information about rasa x
+      - i want to learn more about Rasa X
+      - what is rasa x?
+      - Can you tell me about rasa x?
+      - Tell me about rasa x
+      - tell me what is rasa x
+```
+
+### Responses
+
+Responses for the `ResponseSelector` follow the same naming convention as
+retrieval intents. Besides this, they can have all the characteristics of 
+normal bot [response](domain.mdx#responses). For the FAQ and chitchat intents
+listed above, our responses could look like:
 
-* `config.yml`: Add `ResponseSelector` in your NLU pipeline
+```yaml
+responses:
+  faq/ask_channels: 
+  - text: |
+    We have a comprehensive list of [supported connectors](https://rasa.com/docs/core/connectors/), but if
+    you don't see the one you're looking for, you can always create a custom connector by following
+    [this guide](https://rasa.com/docs/rasa/user-guide/connectors/custom-connectors/).
+  faq/ask_languages:
+  - text: |
+    You can use Rasa to build assistants in any language you want!
+  faq/ask_rasax:
+  - text: |
+    Rasa X is a tool to learn from real conversations and improve your assistant. 
+    Read more [here](https://rasa.com/docs/rasa-x/)
+```
 
-* `domain.yml`: Add a retrieval action `respond_faq` and intent `faq`
+## Putting it all together
 
-* `data/stories.md`: Add a simple story for FAQs
+Once you've done the following, you can train your bot and try it out!
 
-* `test_stories.md`: Add E2E test stories for your FAQs
+- [ ] Added `RulePolicy` to your policies in `config.yml`
+- [ ] Added `ResponseSelector` to your pipeline in `config.yml`
+- [ ] Added at least one rule for responding to FAQ/chitchat
+- [ ] Added examples for your FAQ/chitchat intents
+- [ ] Added responses for your FAQ/chitchat intents
 
-:::
+Even if you have other skills in your bot besides chitchat and FAQs, your bot should be able to respond consistently 
```

Comment:
`Even if you have other skills in your bot` feels a bit clunky. Maybe,

Now, your assistant can respond correctly and consistently to FAQs and chitchat, even if these interjections happen while your assistant is helping the user with another task.